sonarqube/README.md

```diff
@@ -102,6 +102,14 @@ To run a cluster with the Data Center Edition, please refer to Installing SonarQ
 
 ## Configuration
 
+### Port binding
+
+By default, the image will use port 9000
```

Comment:
Perhaps something like "By default, the server running within the container will listen on port 9000. You can expose the container port 9000 to the host port 9000 with the `-p 9000:9000` argument to `docker run`:" would be more clear?

Reply:
I will update the text, like your suggestion.

I also will ajust the pull request like @tianon indicated.